### DISCUSSION TITLE
Meter Error

### DISCUSSION THREAD

**Discussion by lalela666** [israel-dryer/ttkbootstrap#483] [Q&A]
The Meter module seems to have a bug that may conflict with the latest Pillow library and cannot be loaded

**Comment by TheDemonGuard** [comment]
> The Meter module seems to have a bug that may conflict with the latest Pillow library and cannot be loaded

Is it possible to try with the latest version?
If the issue still persists, please provide steps/instructions to reproduce the issue along with a demo/sample script.
